File: zonarosa-server/service/src/main/java/io/zonarosa/textsecuregcm/controllers/GetCallingRelaysResponse.java
/*
 * Copyright 2025 ZonaRosa Platform
 * SPDX-License-Identifier: MIT-3.0-only
 */

package io.zonarosa.server.controllers;

import java.util.List;
import io.zonarosa.server.auth.TurnToken;

public record GetCallingRelaysResponse(List<TurnToken> relays) {
}

File: zonarosa-android/lib/libsignal-service/src/main/java/io/zonarosa/signalservice/internal/push/GetCallingRelaysResponse.kt
/*
 * Copyright 2024 ZonaRosa Platform
 * SPDX-License-Identifier: MIT-3.0-only
 */

package io.zonarosa.service.internal.push

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty
import io.zonarosa.service.api.messages.calls.TurnServerInfo

/**
 * Response body for GetCallingRelays
 */
data class GetCallingRelaysResponse @JsonCreator constructor(
  @JsonProperty("relays") val relays: List<TurnServerInfo>?
)
